packages/ui5-application-writer/templates/optional/loadReuseLibs/webapp/utils/locate-reuse-libs.js

```diff
@@ -1,173 +1,168 @@
 /*eslint-disable semi, no-console*/
 (function (sap) {
-  var fioriToolsGetManifestLibs = function (manifestPath) {
-    var url = manifestPath;
-    var result = "";
-    // SAPUI5 delivered namespaces from https://ui5.sap.com/#/api/sap
-    var ui5Libs = [
-      "sap.apf",
-      "sap.base",
-      "sap.chart",
-      "sap.collaboration",
-      "sap.f",
-      "sap.fe",
-      "sap.fileviewer",
-      "sap.gantt",
-      "sap.landvisz",
-      "sap.m",
-      "sap.ndc",
-      "sap.ovp",
-      "sap.rules",
-      "sap.suite",
-      "sap.tnt",
-      "sap.ui",
-      "sap.uiext",
-      "sap.ushell",
-      "sap.uxap",
-      "sap.viz",
-      "sap.webanalytics",
-      "sap.zen"
-    ];
-    function getKeys(libOrComp,libOrCompKeysString) {
-      Object.keys(libOrComp).forEach(function (libOrCompKey) {
-        // ignore libs or Components that start with SAPUI5 delivered namespaces
-        if (!ui5Libs.some(function (substring) { return libOrCompKey === substring || libOrCompKey.startsWith(substring + "."); })) {
-          if (libOrCompKeysString.length > 0) {
-            libOrCompKeysString = libOrCompKeysString + "," + libOrCompKey;
-          } else {
-            libOrCompKeysString = libOrCompKey;
-          }
-        }
-      });
-      return libOrCompKeysString;
-    }
-    return new Promise(function (resolve, reject) {
-      $.ajax(url)
-        .done(function (manifest) {
-          if (manifest) {
-            if (
-              manifest["sap.ui5"] &&
-              manifest["sap.ui5"].dependencies
-            ) {
-              if (manifest["sap.ui5"].dependencies.libs){
-                result = getKeys(manifest["sap.ui5"].dependencies.libs, result)
-              }
-              if (manifest["sap.ui5"].dependencies.components){
-                result = getKeys(manifest["sap.ui5"].dependencies.components, result)
-              }
-            }
+    const getKeys = function (libOrComp, libOrCompKeysString) {
```

Comment:
We may still need to use ES5 syntax in this file

Reply:
I don't think the UI5 tooling does anything with this file (but I am not sure), however, I also don't understand why we may need ES5 syntax here?

Reply:
Done, thanks 👍